esphomeRemote/esphomeRemote.h

```diff
@@ -649,6 +649,8 @@ void idleTick() {
     return;
   } else if (idleTime > id(sleep_after)) {
     if (!charging) {
+      // deselect light detail to not have a slider active when coming back from sleep
+      lightGroup->lightDetailSelected = false;

```

Comment:
we want this in idleMenu() where we return to the now playing menu and turn off the display if the device isn't charging